addons/pos_mercado_pago/controllers/main.py

```diff
@@ -0,0 +1,92 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+import binascii
+import hashlib
+import hmac
+import logging
+
+from odoo import http
+from odoo.http import request
+
+_logger = logging.getLogger(__name__)
+
+
+class PosMercadoPagoWebhook(http.Controller):
+    @http.route('/pos_mercado_pago/notification', methods=['POST'], type="json", auth="none")
+    def notification(self):
+        """ Process the notification sent by Mercado Pago
+        Notification format is always json
+        """
+        _logger.debug('POST message received on the end point')
+
+        # Check for mandatory keys in header
+        x_request_id = request.httprequest.headers.get('X-Request-Id', None)
+        x_signature = request.httprequest.headers.get('X-Signature', None)
+        if not x_request_id or not x_signature:
+            _logger.debug('POST message received with no X-Request-Id and/or no X-Signature in header')
+            return None
```

Comment:
No, this is not for Mercado, this "return None" and the next one is to return nothing in case of callers other than Mercado want to access this point. The idea is for them to have no response and for us to make the less work possible.
It's very unlikely that Mercado send a bad webhook.